services/motion/builtin/move_request.go

```diff
@@ -612,14 +612,16 @@ func (ms *builtIn) newMoveOnMapRequest(
 		return nil, err
 	}
 
+	req.Obstacles = append(req.Obstacles, octree)
```

Comment:
Is it true that all these obstacles are in the same frame?  Do they need to be?

Reply:
the api message gives us proto geometries which have on way of telling us with what frame they are defined with respect to. It's on the user to know that at call time.